osu.Game/Overlays/SearchableList/SearchableListOverlay.cs

```diff
@@ -125,5 +130,13 @@ protected override void PopOut()
 
             Filter.Search.HoldFocus = false;
         }
+
+        private class PanelContainer : OnlineViewContainer
+        {
+            public PanelContainer(string placeholderMessage)
+                : base(placeholderMessage)
+            {
+            }
+        }
```

Comment:
I thought `OnlineViewContainer` would only be used for inheritance. but after I've given this usage a couple of reads, I would argue that this container should not be abstract at all.